include/glow/Backend/Backend.h

```diff
@@ -98,6 +98,10 @@ class Backend {
   /// performed.
   virtual bool shouldShareBuffers() const { return true; }
 
+  /// Modify the \p optimizationOpts however desired.
+  virtual void
+  setOptimizationOptions(OptimizationOptions &optimizationOpts) const {};
```

Comment:
might be for future PRs, can we consolidate buffer sharing support in the OptimizationOptions